tests/Unit/Domain/Test_DomainHelpers.cpp

```diff
@@ -239,7 +239,8 @@ void test_wedge_map_generation_against_domain_helpers(
   const auto maps = wedge_coordinate_maps<Frame::Inertial>(
```

Comment:
What is up with these extra lines in your commit messages?

Reply:
I misinterpreted the extra line between the commit "title" and the rest of the commit message I've seen on other people's commits as a sign that I should use blank lines on every other line. I'll fix it :+1: